packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -623,18 +625,24 @@ class _ResidentWebRunner extends ResidentWebRunner {
         '// Flutter web bootstrap script for $importedEntrypoint.',
         '',
         "import 'dart:ui' as ui;",
+        "import 'dart:async';",
         '',
         "import '$importedEntrypoint' as entrypoint;",
         if (hasWebPlugins)
           "import 'package:flutter_web_plugins/flutter_web_plugins.dart';",
         if (hasWebPlugins)
           "import '$generatedImport';",
         '',
+        'typedef _UnaryFunction = Function(List<String> args);',
+        'typedef _NullaryFunction = Function();',
```

Comment:
Should these have return types?

Reply:
Technically these could return null, void, Future<x>. This is the pattern used by the engine already

Reply:
I guess we're not linting generated code. We could make them return `dynamic` for explicitness.

Reply:
done